bpf-prog/ipv6-healthprobe-tc/cmd/ipv6-healthprobe-tc/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"net"
+
+	"github.com/Azure/azure-container-networking/bpf-prog/ipv6-healthprobe-tc/pkg/egress"
+	"github.com/Azure/azure-container-networking/bpf-prog/ipv6-healthprobe-tc/pkg/ingress"
+	"github.com/vishvananda/netlink"
+
+	"github.com/cilium/ebpf/rlimit"
+	"go.uber.org/zap"
+)
+
+var logger *zap.Logger
+
+func main() {
+	// Set up logger
+	logger, _ = zap.NewProduction()
```

Comment:
This won't write to a file right? Can we use something similar to what we have for cni: https://github.com/Azure/azure-container-networking/blob/4370e6e93c7623a70446580417df1ab0fc2b4ff9/cni/log/logger.go#L31

Reply:
These logs will just add a few lines to the container on startup, and if any error occurs those will show as well. From my understanding NewProductionEncoderConfig is to allow us to customize the log format. NewProduction comes with a preset encoder config. Do we need to customize log format in this case?

Reply:
My concern is where will this get logged in filesystem ? Are we specifying any file to which the logs will be written. If it is, then we can ignore the comment.

Reply:
ah ok, updated this to write to ipv6-healthprobe-tc.log